architecture/cicd_gitops.adoc

```diff
@@ -0,0 +1,62 @@
+
+CI/CD introduces ongoing automation and continuous monitoring throughout the lifecycle of applications, from integration and testing phases to delivery and deployment. CI/CD is a process that is often referred to as a "CI/CD pipeline" that adds a high degree of recurring automation and continuous monitoring to application development.
+
+[id="cicd_admin"]
+== CI/CD for cluster administration and application configuration management
+
+_Continuous integration_ is an automation process for developers. Code changes to an application are regularly built, tested, and merged to a shared repository.
+
+_Continuous delivery_ and _continuous deployment_ are closely related concepts that are sometimes used interchangeably that refer to automation of the pipeline.
+Continuous delivery uses automation to ensure that a developer’s changes to an application are tested and sent to a repository, where an operation team can deploy them to a production environment. Continuous deployment enables the release of changes, starting from the repository until production. Continuous deployment prevents the operations team from getting overloaded and speeds up application delivery.
+
+[id="cicd_gitops_methodology"]
+== The GitOps methodology and practice
+
+_GitOps_ is a set of practices that use Git pull requests to manage infrastructure and application configurations. The Git repository in GitOps is the only source of truth and contains the entire state of the system so that the trail of changes to the system state are visible and auditable. GitOps enables you to implement a DevOps methodology.
+
+You can use GitOps tooling to create repeatable and predictable processes for managing and recreating {product-title} clusters and applications. By using GitOps, you can address the issues of infrastructure and application configurations sprawl. It simplifies the propagation of infrastructure and application configurations changes across multiple clusters by defining your infrastructure and applications definitions as “code”. Implementing GitOps for your cluster configuration files can make automated installation easier and allow you to configure automated cluster customizations. You can apply the core principles of developing and maintaining software in a Git repo to the creation and management of your cluster and application configuration files.
+
+Automating both your cluster configuration and container development process by using GitOps allows you to pick and choose where and when to adopt this practice, using {product-title}. Using a CI pipeline that pairs with your GitOps strategy and execution plan is ideal. {product-title} provides the flexibility to choose when and how you integrate this methodology into your business practices and pipelines.
```

Comment:
"adopt this practice" - What practice? The GitOps practice?
"using OCP" sounds oddly attached here. Is your point that OCP integrates GitOps or that GitOps makes OCP easier? Not really sure.

"Using a CI pipeline that pairs with your GitOps strategy and execution plan is ideal." Why? I don't think this sentence strengthens your argument, which (IIUC) is that OCP's flexibility lets you integrate CI/CD for better business practices.